libs/ui-toolkit/src/components/copy-with-tooltip/copy-with-tooltip.spec.tsx

```diff
@@ -0,0 +1,24 @@
+import {
+  fireEvent,
+  render,
+  screen,
+  waitFor,
+  waitForElementToBeRemoved,
+} from '@testing-library/react';
+import { CopyWithTooltip, TOOLTIP_TIMEOUT } from './copy-with-tooltip';
+
+jest.useFakeTimers();
```

Comment:
question (blocking): why do we need to fake timers here?

Reply:
Oh I see, can we move this to a test or a before each jest.useRealTimers() at the end or in an after each

Reply:
Hm just checking the docs on this and it says its a 'global operation and will affect other tests within the same file' that said its probably best to isolate it

Reply:
Moved to beforeAll and restored in afterAll